### ISSUE TITLE
DL3042 reports pipenv, however pipenv doesn't seem to support the option

### ISSUE THREAD

**Issue by Majkl578** [hadolint/hadolint#511]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Only report `pip`.

### Actual behavior

`pipenv install` is reported by DL3042.

### Steps to reproduce the behavior

**Dockerfile**

```Dockerfile
FROM scratch

RUN pipenv install
```

Hadolint v1.19.0-45-gef91156-alpine
